library/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/processor/WireframeUtilsTest.kt

```diff
@@ -239,7 +239,7 @@ internal class WireframeUtilsTest {
 
     @ParameterizedTest
     @MethodSource("coverAllWireframes")
-    fun `M return false W checkIsValidWireframe(){ covered by another with solid background }`(
+    fun `M return true W checkIsValidWireframe(){ covered by another with solid background }`(
```

Comment:
we need to rename the tests in this class, there is no method called `checkIsValidWireframe` anymore and this test in particular checks `checkWireframeIsCovered`